### PR TITLE
Add release notes for v0.17.4, bump version.

### PR DESCRIPTION
**Pull request by martinkim0**
Addresses #1688.

### CODE REVIEW

File: docs/release_notes/v0.17.4.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# New in 0.17.4 (2021-09-19)
```
Comment (adamgayoso) [inline docs/release_notes/v0.17.4.md:1]:
```suggestion
# New in 0.17.4 (2021-09-20)
```

File: docs/release_notes/v0.17.4.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Changes
+- Update to PyTorch Lightning 1.7 [#1622].
+- Update publication links in `README` [#1667].
+- Change to `super().__init__()` [#1671].
```
Comment (adamgayoso) [inline docs/release_notes/v0.17.4.md:6]:
```suggestion
- Remove instances of `super().__init__()` with an argument in `super()` [#1671].
```

File: docs/release_notes/v0.17.4.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Add linkcode, hoverxref, and grid to docs [#1678].
+
+## Bug Fixes
+- Fix likelihood parameters in `RNASeqMixin` [#1618].
```
Comment (adamgayoso) [inline docs/release_notes/v0.17.4.md:10]: I know you weren't here for most of these, but I would add specifically what this fixes, fix `get...` when gene likelihood is not zinb (as an example)

**Review by adamgayoso — CHANGES_REQUESTED** [review]
Thanks! Can you also add the last few PRs I added quickly as bug fixes?

File: docs/release_notes/v0.17.4.md (hunk 3 above)
Comment (martinkim0) [reply docs/release_notes/v0.17.4.md:10]: Got it! I wasn't sure how much detail to include here - I'll take a closer look at the diffs from now on then.

File: docs/release_notes/v0.17.4.md (hunk 2 above)
Comment (martinkim0) [reply docs/release_notes/v0.17.4.md]: Resolved in new commit

File: docs/release_notes/v0.17.4.md (hunk 1 above)
Comment (martinkim0) [reply docs/release_notes/v0.17.4.md]: Resolved in new commit.

**Review by adamgayoso — APPROVED** [review]

**Pull request merged**